fv3core/fv3core/stencils/dyn_core.py

```diff
@@ -525,6 +525,86 @@ def __init__(
             self.comm, grid_indexing, backend=stencil_factory.backend
         )
 
+    def _checkpoint_csw_in(self, state):
```

Comment:
I see that you use a tag `in/out` for `_checkpoint_fvdynamics`. Is there a reason you cannot do them for `c_sw` and `d_sw`? It looks to me that the variables are the same.

Reply:
No reason. Updated them to use the same "tag" style.